localization/english/dvg_names_l_english.yml

```diff
@@ -34,6 +35,8 @@
  Alexey:0 "Alexey"
  Bronislav:0 "Bronislav"
  Dimitriy:0 "Dimitriy"
+ Selim:0 "Selim"
```

Comment:
These already exist in vanilla files, so it can be removed
[07:17:51][pdx_localize.cpp:279]: Duplicate localization key. Key 'Murad' is defined in both 'localization/english/dvg_names_l_english.yml' and 'localization/english/names_l_english.yml'.
[07:17:51][pdx_localize.cpp:279]: Duplicate localization key. Key 'Selim' is defined in both 'localization/english/dvg_names_l_english.yml' and 'localization/english/names_l_english.yml'.